internal/kafka/confluent_kafka.go

```diff
@@ -127,6 +127,7 @@ func retrieveUnsecuredToken(e ckafka.OAuthBearerTokenRefresh, tokenValue string)
 func newProducer(kafka *config.KafkaClusterConfig, clientID, configPath string, configStrings []string) (*ckafka.Producer, error) {
 	configMap, err := getProducerConfigMap(kafka, clientID)
 	if err != nil {
+		log.CliLogger.Warn("Failed to get Confluent Kafka producer configuration map.")
 		return nil, err
```

Comment:
Should we wrap these into the same error?

Reply:
Something like `failed to get producer configuration: %w`

Reply:
The error will later be returned and shown as "failed to create producer/consumer"

Reply:
maybe just "failed to get configuration" in that case?